run.sh

```diff
@@ -91,6 +108,8 @@ args=(
   --rpc-drone-address 127.0.0.1:9900
   --accounts "$dataDir"/accounts
   --log -
+  --enable-rpc-exit
+  --init-complete-file "$dataDir"/init-completed
```

Comment:
Specifically needed for `run-sanity.sh`; Enable unconditionally because doing so adds no harm, I think.